src/NamespacedValue.ts

```diff
@@ -109,10 +109,10 @@ export class UnstableValue<S extends string, U extends string> extends Namespace
     }
 
     public get name(): U {
-        return this.unstable;
+        return this.unstable!;
```

Comment:
Should be `U | undefined` return type rather than non-null assertion

Reply:
The constructor says

```
// Stable is optional, but one of the two parameters is required, hence the weird-looking types.
// Goal is to to have developers explicitly say there is no stable value (if applicable).
```

So we can assert it will be defined given the `stable` wasn't. Will try express it with an overload